sdk/provisioning/Azure.Provisioning/src/Generated/Models/ManagedServiceIdentityType.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+using System.Runtime.Serialization;
+
+namespace Azure.Provisioning.Resources;
+
+/// <summary>
+/// Type of managed service identity (where both SystemAssigned and
+/// UserAssigned types are allowed).
+/// </summary>
+public enum ManagedServiceIdentityType
+{
+    /// <summary>
+    /// None.
+    /// </summary>
+    None,
+
+    /// <summary>
+    /// SystemAssigned.
+    /// </summary>
+    SystemAssigned,
+
+    /// <summary>
+    /// UserAssigned.
+    /// </summary>
+    UserAssigned,
+
+    /// <summary>
+    /// SystemAssigned, UserAssigned.
+    /// </summary>
+    [DataMember(Name = "SystemAssigned, UserAssigned")]
+    SystemAssignedUserAssigned,
+}
```

Comment:
And we've got plain enums like `ManagedServiceIdentityType` for good measure.   Since `BicepValue<T>` can always allow a `StringLiteral` expression and everything's "an input" (with respect to forward compat), we're able to eschew extensible enums.